lib/ts/recipe/oauth2provider/types.ts

```diff
@@ -446,6 +448,38 @@ export type APIInterface = {
               options: APIOptions;
               userContext: UserContext;
           }) => Promise<InstrospectTokenResponse | GeneralErrorResponse>);
+    endSessionGET:
+        | undefined
+        | ((input: {
+              params: Record<string, string>;
+              session?: SessionContainerInterface;
+              options: APIOptions;
+              userContext: UserContext;
+          }) => Promise<{ redirectTo: string }>);
+    endSessionPOST:
+        | undefined
+        | ((input: {
+              params: Record<string, string>;
+              session?: SessionContainerInterface;
+              options: APIOptions;
+              userContext: UserContext;
+          }) => Promise<{ redirectTo: string }>);
+    logoutGET:
+        | undefined
+        | ((input: {
+              logoutChallenge: string;
+              options: APIOptions;
+              session?: SessionContainerInterface;
+              userContext: UserContext;
+          }) => Promise<{ redirectTo: string }>);
+    logoutPOST:
+        | undefined
+        | ((input: {
+              logoutChallenge: string;
+              options: APIOptions;
+              session?: SessionContainerInterface;
+              userContext: UserContext;
+          }) => Promise<{ redirectTo: string }>);
```

Comment:
Why do we need 2 versions for this?

Reply:
We didn't need `logoutGET` as an API because it was only called internally. Removed it.